packages/inter-protocol/src/auction/params.js

```diff
@@ -0,0 +1,212 @@
+import {
+  CONTRACT_ELECTORATE,
+  makeParamManager,
+  ParamTypes,
+} from '@agoric/governance';
+import { TimeMath, RelativeTimeRecordShape } from '@agoric/time';
+import { M } from '@agoric/store';
+
+/** @typedef {import('@agoric/governance/src/contractGovernance/typedParamManager.js').AsyncSpecTuple} AsyncSpecTuple */
+/** @typedef {import('@agoric/governance/src/contractGovernance/typedParamManager.js').SyncSpecTuple} SyncSpecTuple */
+
+// TODO duplicated with zoe/src/TypeGuards.js
+export const InvitationShape = M.remotable('Invitation');
+
+/**
+ * The auction will start at AUCTION_START_DELAY seconds after a multiple of
+ * START_FREQUENCY, with the price at STARTING_RATE. Every CLOCK_STEP, the price
+ * will be reduced by DISCOUNT_STEP, as long as the rate is at or above
+ * LOWEST_RATE, or until START_FREQUENCY has elapsed. in seconds, how often to
+ * start an auction
+ */
+export const START_FREQUENCY = 'StartFrequency';
+/** in seconds, how often to reduce the price */
+export const CLOCK_STEP = 'ClockStep';
+/** discount or markup for starting price in basis points. 9999 = 1bp discount */
+export const STARTING_RATE = 'StartingRate';
```

Comment:
this comment won't appear when setting this params in UIs.  WDYT of appending 'BP' for all the basis points values?

alternately, express as a ratio? if it's too cumbersome to include the brands I think we should consider a RATIONAL param type because the work-around of a Nat is worse